Uplift/Views/ClassDetail/ClassDetailHeaderView.swift

```diff
@@ -9,20 +9,26 @@
 import UIKit
 
 protocol ClassDetailHeaderViewDelegate: class {
+    func classDetailHeaderViewFavoriteButtonTapped()
+    func classDetailHeaderViewBackButtonPressed()
     func classDetailHeaderViewLocationSelected()
     func classDetailHeaderViewInstructorSelected()
+    func classDetailHeaderViewShareButtonTapped()
```

Comment:
Move `classDetailHeaderViewShareButtonTapped` above  `classDetailHeaderViewLocationSelected`

Reply:
I'm going to organize these alphabetically - curious as to why you recommend putting `classDetailHeaderViewShareButtonTapped` above `classDetailHeaderViewLocationSelected`?